bin/build_container_image

```diff
@@ -3,7 +3,11 @@
 REGISTRY=${REGISTRY:-"docker.io"}
 ORGANIZATION=${ORGANIZATION:-"manageiq"}
 DEFAULT_TAG="latest"
-[[ $TRAVIS_BRANCH != "master" ]] && DEFAULT_TAG="$DEFAULT_TAG-$TRAVIS_BRANCH"
+if [[ "$GITHUB_EVENT_NAME" == "pull_request" ]]; then
+  DEFAULT_TAG="$DEFAULT_TAG-pr"
```

Comment:
@bdunne Thoughts here?  SInce the actual name doesn't matter, do we even care about this?  I was thinking we could just leave it as "latest"

Reply:
I noticed, previously that if run locally this would generate `latest-` as the name, because TRAVIS_BRANCH was blank  (actually it would probably blow up if run locally)...thinking I might fix that local use case as well

Reply:
I updated with this and fixed the `latest-` thing.  Also fixed it so that branches with `/` will work (even though that doesn't effect master, it could affect pushes to forks, where the fork has the push event.

Reply:
This PR is an example where it will tag as latest: https://github.com/ManageIQ/manageiq-rpm_build/runs/4587369238?check_suite_focus=true#step:4:3079

Reply:
I'm hesitant to add `GITHUB_EVENT_NAME`, maybe we can get the ref from git itself?

Reply:
I can try, but we need the even to know if it's a PR or a push build...not sure how you get that out of git.  Basically, we need push builds that are not master (i.e. morphy, lasker, etc.)